### PR TITLE
For #33290, added missing icon

### PR DESCRIPTION
**Pull request by jfboismenu**
Replaced the default icons with a missing icon for the header and outputs. When core releases support for non-existing path for config_path settings, these will show up.

### CODE REVIEW

File: resources/build_resources.sh [hunk 1]
```diff
@@ -13,6 +13,9 @@
 # The path to output all built .py files to: 
 UI_PYTHON_PATH=../python/tk_multi_publish/ui
 
+# The path to where the PySide binaries are installed
+PYTHON_BASE="/Applications/Shotgun.app/Contents/Resources/Python"
```
Comment (josh-t) [inline resources/build_resources.sh]: I'm guessing we shouldn't hardcode this here. Should this be handled by the developer in whatever env they're working in? i.e. making sure `pyside-uic` and `pyside-rcc` are in their path? Maybe just a comment instead?

**Comment by manneohrstrom** [comment]
ok, so we already had default icons to fall back on!

File: resources/build_resources.sh (hunk 1 above)
Comment (jfboismenu) [inline resources/build_resources.sh]: Maybe I am wrong, but didn't we say recently that we should always use the Desktop's Python to build our resources? I'm putting on Hot Topics.

Comment (manneohrstrom) [inline resources/build_resources.sh]: +1 not to hard code this. IMHO Seems we are overcomplicating things a bit. @josh-t suggestion to put add a comment about it seems better. And if we do update it, we should probably not do it in a single app, but instead look at if we can centralize this build script that is present in so many apps. or do a blanket update.

**Pull request closed without merging**